src/routing_utils/nats_client/main.go

```diff
@@ -14,30 +19,41 @@ import (
 )
 
 const USAGE = `Usage:
-/var/vcap/jobs/gorouter/bin/nats_client [COMMAND] [SUBJECT] [MESSAGE]
+/var/vcap/jobs/gorouter/bin/nats_client <gorouter.yml> <COMMAND>
 
 COMMANDS:
-  subscribe    (Default) Streams NATS messages from server with provided SUBJECT. Default SUBJECT is 'router.*'
-               Example: /var/vcap/jobs/gorouter/bin/nats_client subscribe 'router.*'
+  sub          [SUBJECT] [MESSAGE] 
+               (Default) Streams NATS messages from server with provided SUBJECT. Default SUBJECT is 'router.*'
+               Example: /var/vcap/jobs/gorouter/bin/nats_client sub 'router.*'
 
-  publish      Publish the provided message JSON to SUBJECT subscription. SUBJECT and MESSAGE are required
-               Example: /var/vcap/jobs/gorouter/bin/nats_client publish router.register '{"host":"172.217.6.68","port":80,"uris":["bar.example.com"]}'
+  pub          [SUBJECT] [MESSAGE]
+               Publish the provided message JSON to SUBJECT subscription. SUBJECT and MESSAGE are required
+               Example: /var/vcap/jobs/gorouter/bin/nats_client pub router.register '{"host":"172.217.6.68","port":80,"uris":["bar.example.com"]}'
```

Comment:
The example is not correct. There should be something like placeholder for gorouter.yml before "pub".

Reply:
you are right! fixing now

Reply:
Actually, I just learned that the original help text was indeed correct. The reason is that there are two `nats_client` commands:
- `/var/vcap/packages/routing_utils/bin/nats_client` (this file)
- `/var/vcap/jobs/gorouter/bin/nats_client` (a script that calls this file, implicitly adding `/var/vcap/jobs/gorouter/config/gorouter.yml` as first argument

See [the script template](https://github.com/cloudfoundry/routing-release/blob/develop/jobs/gorouter/templates/nats_client.erb)

So I've reverted the help text to only include the new commands but no changes to gorouter.yml being needed.